package-lock.json

```diff
@@ -10672,6 +10672,19 @@
       "resolved": "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz",
```

Comment:
shouldn't be any changes here

Reply:
I'm not sure why there is a change here...I assume because I installed and then uninstalled that gatsby plugin?